.github/workflows/prebuilt_binaries.yml

```diff
@@ -0,0 +1,399 @@
+            elif [ -n "$registry_cache_image" ]; then
+              echo "$registry_cache_image:$(echo ${{ github.ref_name }} | tr / -)"
+            fi
+            echo multiline
+          } >> $GITHUB_OUTPUT
+
+      - name: Set up context for buildx
+        run: |
+          docker context create builder_context
+
+      - name: Set up buildx runner
+        uses: docker/setup-buildx-action@v2
+        with:
+          endpoint: builder_context
+          driver-opts: network=host
+
+      - name: Extract metadata
+        id: metadata
+        uses: docker/metadata-action@v5
+        with:
+          images: ${{ steps.config.outputs.image_name }}
+          flavor: |
+            latest=false
+            prefix=${{ steps.config.outputs.tag_prefix }},onlatest=true
+          tags: |
+            type=ref,enable=${{ steps.config.outputs.custom_tags == '' }},event=branch
+            type=ref,enable=true,prefix=${{ steps.config.outputs.tag_prefix }}pr-,event=pr
+            type=ref,enable=true,event=tag
+            ${{ steps.config.outputs.custom_tags }}
+          labels: |
+            org.opencontainers.image.title=cuda-quantum-assets
+            org.opencontainers.image.description=Pre-built Linux binaries for CUDA Quantum
+
+      - name: Build assets
+        id: docker_build
+        uses: docker/build-push-action@v5
+        with:
+          context: .
+          file: ./docker/build/assets.Dockerfile
+          build-args: |
+            release_version=${{ steps.config.outputs.cudaq_version }}
+            base_image=${{ inputs.platform_base_image }}
+          tags: ${{ steps.metadata.outputs.tags }}
+          labels: ${{ steps.metadata.outputs.labels }}
+          platforms: ${{ inputs.platform }}
+          cache-from: |
+            ${{ inputs.build_cache }}
+            ${{ steps.config.outputs.additional_build_caches }}
+          push: true
+
+      - name: Install Cosign
+        if: inputs.environment
+        uses: sigstore/cosign-installer@v3.1.1
+
+      - name: Sign image with GitHub OIDC Token
+        if: inputs.environment
+        env:
+          DIGEST: ${{ steps.docker_build.outputs.digest }}
+          TAGS: ${{ steps.metadata.outputs.tags }}
+        run: cosign sign --yes --recursive "${TAGS}@${DIGEST}"
+
+      - name: Build installer
+        uses: docker/build-push-action@v5
+        with:
+          context: .
+          file: ./docker/release/installer.Dockerfile
+          build-args: |
+            release_version=${{ steps.config.outputs.cudaq_version }}
+            base_image=${{ steps.config.outputs.image_name }}@${{ steps.docker_build.outputs.digest }}
+          platforms: ${{ inputs.platform }}
+          outputs: type=local,dest=/tmp/install
+
+      - name: Upload installer
+        uses: actions/upload-artifact@v4
+        with:
+          name: ${{ steps.config.outputs.artifact_name }}
+          path: /tmp/install
+          retention-days: 5
```

Comment:
It looks like the artifacts are ~620MB per platform (~1.4 GB per CI run). Hopefully we don't run out of space? (I don't know if we have an artifact quota or not.)

Reply:
There is a quota limit, and I added a clean up that removes the artifacts after validation.